docs/reference/connector/apis/update-connector-filtering-api.asciidoc

```diff
@@ -0,0 +1,186 @@
+[[update-connector-filtering-api]]
+=== Update connector filtering API
+
+preview::[]
+
+++++
+<titleabbrev>Update connector filtering</titleabbrev>
+++++
+
+Updates the `filtering` configuration of a connector.
+
+[[update-connector-filtering-api-request]]
+==== {api-request-title}
+
+`PUT _connector/<connector_id>/_filtering`
+
+[[update-connector-filtering-api-prereq]]
+==== {api-prereq-title}
+
+* To sync data using connectors, it's essential to have the Elastic connectors service running.
+* The `connector_id` parameter should reference an existing connector.
+
+[[update-connector-filtering-api-path-params]]
+==== {api-path-parms-title}
+
+`<connector_id>`::
+(Required, string)
+
+[role="child_attributes"]
+[[update-connector-filtering-api-request-body]]
+==== {api-request-body-title}
+
+`filtering`::
+(Required, array) The filtering configuration for the connector. This configuration determines the set of rules applied for filtering data during syncs.
+
+Each entry in the `filtering` array represents a set of filtering rules for a specific data domain and includes the following attributes:
+
+- `domain` (Required, string) +
+Specifies the data domain to which these filtering rules apply.
+
+- `active` (Required, object) +
```

Comment:
@leemthompo I used `-` to mark different indentation level vs. `*` - it works as expected, please let me know if there is a different best practice regarding lists with multiple indentation levels. (I remember you suggested changing `-` to `*` in other PR for a different reason I think)

Reply:
This seems to work fine, handwriting these docs is a nightmare for nested stuff, hopefully will have autogen from spec in the not too distant future

Reply:
You can always check the [asciidoctor docs](https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/) if in doubt